### PR TITLE
Add entry to JO list, remove CRLF line terminators

### PR DESCRIPTION
**Pull request by anadahz**

### CODE REVIEW

File: lists/jo.csv [hunk 1]
```diff
@@ -1,186 +1,187 @@
...
+http://www.thejordantoday.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.tourismworld-seyaha.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.wahanews.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.wehdanews.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.wejhat.net,NEWS,News Media,2014-04-15,citizenlab,
+http://www.wtn24.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.yarmouknews.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.yawmak.com,NEWS,News Media,2014-04-15,citizenlab,
+http://www.zarqanews.net,NEWS,News Media,2014-04-15,citizenlab,
+https://www.mykalimag.com,CULTR,Culture,2018-04-12,https://storage.googleapis.com/qurium/index.html,Blocked since July 2016
```
Comment (hellais) [inline lists/jo.csv]: Missing a newline here.

Comment (hellais) [reply lists/jo.csv]: Ah I see what you did, the https://storage.googleapi.com/quirium/index.htlm is the authorname. Can you please replace the author name with your name or `anon` if you don't want to be identified, and put the url in the comment section?

Comment (anadahz) [reply lists/jo.csv]: @hellais Thanks for reviewing the PR.
I made the proposed changes.

**Comment by hellais** [comment]
lgtm

**Pull request merged**